test/integration/services/data/test-duplicate-claim-check.js

```diff
@@ -0,0 +1,76 @@
+var expect = require('chai').expect
+var databaseHelper = require('../../../helpers/database-setup-for-tests')
+
+var duplicateClaimCheck = require('../../../../app/services/data/duplicate-claim-check')
+var reference1 = 'V123456'
+var reference2 = 'V234567'
+var reference3 = 'V345678'
```

Comment:
use `const REFERENCE1` etc.

const and uppercase makes it more obvious that the value will not be updated